zppy/global_time_series.py

```diff
@@ -52,6 +53,48 @@ def global_time_series(config, scriptDir, existing_bundles, job_ids_file):
             if skip:
                 continue
 
+            # Handle legacy parameter
+            if (not c["plots_original"]) and c["plot_names"]:
```

Comment:
The original parameter name for which plots you want to display was "plot_names". Given the new by-component plotting, I figured `plots_original` was a better name. 

Question 2) Is `plots_original` a sufficient name for the parameter? `plot_names` will continue to be usable, but will be deprecated.

Reply:
I have no concerns on your suggestion.